build/repo.targets

```diff
@@ -1,8 +1,12 @@
 <Project>
+  <!--
+    TODO include VSIX build in default lifecycle builds
```

Comment:
I chose not to as we only have one CI agent with compatible build tools installed.